tox.ini

```diff
@@ -4,10 +4,10 @@ envlist =
     lint
     docs
     packaging
-    py{36,37,38,39}-2.9
-    py{36,37,38,39}-2.10
-    py{36,37,38,39}-2.11
-    py{36,37,38,39}-devel
+    py{38,39}-2.9
```

Comment:
ansible 2.9.x and ansible-base 2.10.x are EOL and should probably also be removed.